ledger/participant-state/kvutils/src/test/lib/scala/com/daml/ledger/participant/state/kvutils/ParticipantStateIntegrationSpecBase.scala

```diff
@@ -713,11 +713,17 @@ object ParticipantStateIntegrationSpecBase {
   private val participantId: ParticipantId = Ref.ParticipantId.assertFromString("test-participant")
   private val sourceDescription = Some("provided by test")
 
-  val archives = {
+  private val archives = {
     val reader = DarReader { (_, stream) => Try(DamlLf.Archive.parseFrom(stream)) }
-    val fileName = new File(rlocation(com.daml.ledger.test.TestDars.fileNames("model")))
-    val Success(testDar) = reader.readArchiveFromFile(fileName)
-    testDar.all
+    val fileName = com.daml.ledger.test.TestDars.fileNames("model")
+    reader
+      .readArchive(
+        fileName,
+        new ZipInputStream(this.getClass.getClassLoader.getResourceAsStream(fileName)),
```

Comment:
This change allowed to reduce the number of published libraries

Reply:
Much better, thanks!